### ISSUE TITLE
Migrate Prints and Drawings Collection

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#1956]
**Story**
Migrate the Prints and Drawings Collection (PID 56) from FindIt into DCS.

**Acceptance**

- [x]  Export image files from Fedora
- [x]  Import image files into the DCS pairtree
- [x]  Generate list of Parent OIDs
- [x]  Import the objects into DCS UAT into the Med admin set
- [x]  Audit objects in DCS UAT with collection owner
- [x]  Import the objects into DCS PROD into the the Med admin set
- [x]  Audit objects in DCS PROD
- [x]  Update handles URLs to point to DCS

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm 1828 simple objects in uat. https://collections-uat.library.yale.edu/management/batch_processes/833 One object  17339580 does not have image in Fedora or ladybird.

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm Here is the handle:
[https://app.zenhub.com/files/242185115/057e235d-7887-4398-b67b-2590eab0f4b0/download](https://app.zenhub.com/files/242185115/057e235d-7887-4398-b67b-2590eab0f4b0/download)

**Comment by sshetenhelm** [comment]
I wrote to Medical to see if they can give us the missing image and am waiting on a response.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale  they would like us to please import everything to PROD so it is available. They will provide the missing file as soon as they can. 

Note to self: had to run batch process to update visibility from private to public.

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm @motropuk 1828 simple objects were ingested into prod. One object 17339580 does not have image in Fedora or ladybird.
During the batch process, an extra child was created on three objects, e.g. 17338856. I could delete the extra child to fix it, however It looks that it is a new issue. I am checking the log to trace the cause of the issue.

**Comment by motropuk** [comment]
@MaggieZhaoYale I think its fine to remove the extra children for 17340024, 17338856, 17339082 and 17338848 in Prod. Looks the they all have correct images that came over and either the extra child does not actually display once you go into the parent in the management app, or its a clear duplicate. As 17339580, sounds like @sshetenhelm is still waiting on Medical for them to provide this

**Comment by MaggieZhaoYale** [comment]
@motropuk Thank you. The extra children were deleted.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale all we need to do is add the outstanding image I send you via Box, and then we are good to go! :)

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm add the missing image and re-processed https://collections.library.yale.edu/management/parent_objects/17339580
